### PR TITLE
Created new post-mortem document.

### PR DESCRIPTION
**Pull request by pmaslana**

### CODE REVIEW

File: 2022-01/2022-01-20-EmployeeCompromise.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+IPS team: Infrastructure, Platforms, and Security members who responded to the potential incident.
+
+## Timeline (all times UTC )
+
+Date/Time Stamps (Captured from keybase messages)
+
+| Day | Date (2021) | Time (UTC) | Event |
+|-----|-------------|-----------|-------|
+| Thur | Jan 20 | 22:40 | Affected employee logged out of Google Workspace account and received a prompt that his account was disabled, upon attempting to log back in.
+| Thur | Jan 20 | 22:06 | IPS team disables users Okta accounts and searched for abnormal activity. No abnormal login activity for affected employees.
```
Comment (cmmarslender) [inline 2022-01/2022-01-20-EmployeeCompromise.md:25]: Affected employee's okta acount was not disabled, only Admin employee

File: 2022-01/2022-01-20-EmployeeCompromise.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| Thur | Jan 20 | 22:40 | Aside from the 'cookies reset and forced re-login' and 'Account suspended' actions, no other abnormal activity exists.
+| Thur | Jan 20 | 23:15 | Zoom call with Incident Response Support AWS team. The AWS team suggested verifying no abnormal activity exists in CloudTrail for Chia’s two AWS accounts. They made the suggestion to create a SEV2 ticket and send an email to abuse@amazonaws.com for further troubleshooting. 
+| Thur | Jan 20 | 23:17 | No abnormal activity was found in CloudTrail.
+| Thur | Jan 20 | 23:20 | AWS SEV2 support was requested.
+| Thur | Jan 20 | 23:22 | AWS Abuse case was created.
+| Thur | Jan 20 | 23:55 | The admin employee joined a zoom meeting with IPS to discuss the current situation and details.
+| Fri | Jan 21 | 00:20 | Google Admin support chat was initiated for further investigation.
+| Fri | Jan 21 | 01:31 | Cleared all active sessions and third party integrations from admin users' Google account, enabling access to his calendar and email.
+| Fri | Jan 21 | 01:34 | Enabled admin Google account only, leaving his Okta account disabled.
```
Comment (cmmarslender) [inline 2022-01/2022-01-20-EmployeeCompromise.md:38]:
```suggestion
| Fri | Jan 21 | 01:34 | Enabled admin employee's Google account only, leaving his Okta account disabled.
```

File: 2022-01/2022-01-20-EmployeeCompromise.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+| Thur | Jan 20 | 22:23 | The IPS team coordinated with AWS technical support chat to identify any information related to the AWS IP address.
+| Thur | Jan 20 | 22:40 | Aside from the 'cookies reset and forced re-login' and 'Account suspended' actions, no other abnormal activity exists.
+| Thur | Jan 20 | 23:15 | Zoom call with Incident Response Support AWS team. The AWS team suggested verifying no abnormal activity exists in CloudTrail for Chia’s two AWS accounts. They made the suggestion to create a SEV2 ticket and send an email to abuse@amazonaws.com for further troubleshooting. 
+| Thur | Jan 20 | 23:17 | No abnormal activity was found in CloudTrail.
+| Thur | Jan 20 | 23:20 | AWS SEV2 support was requested.
+| Thur | Jan 20 | 23:22 | AWS Abuse case was created.
+| Thur | Jan 20 | 23:55 | The admin employee joined a zoom meeting with IPS to discuss the current situation and details.
+| Fri | Jan 21 | 00:20 | Google Admin support chat was initiated for further investigation.
+| Fri | Jan 21 | 01:31 | Cleared all active sessions and third party integrations from admin users' Google account, enabling access to his calendar and email.
```
Comment (cmmarslender) [inline 2022-01/2022-01-20-EmployeeCompromise.md:37]:
```suggestion
| Fri | Jan 21 | 01:31 | Cleared all active sessions and third party integrations from admin users' Google account, enabling access to his calendar by other employees and allowing email to be received in the inbox for when the okta account was reenabled.
```

File: 2022-01/2022-01-20-EmployeeCompromise.md (hunk 2 above)
Comment (cmmarslender) [reply 2022-01/2022-01-20-EmployeeCompromise.md:38]: Do we mention anywhere that the admin role was removed from this google account?

File: 2022-01/2022-01-20-EmployeeCompromise.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| Fri | Jan 21 | 01:34 | Enabled admin Google account only, leaving his Okta account disabled.
+| Fri | Jan 21 | 16:49 | Admin employee's Okta account was enabled.
+| Fri | Jan 21 | 17:02 | Affected employee was Google account was enabled. 
+| Fri | Jan 21 | 18:20 | The admin employee mentioned that the cause of the account suspension may have been due to a Rippling integration. The account may have been automatically disabled when the affected employee's information was changed.
+| Fri | Jan 21 | 23:01 | IPS reached out to Rippling to discuss situation. They confirmed that the affected employee's account was suspended by Rippling, due to account information being changed and the application being integrated with the admin employee's Google Workspace account. 
+
+## Root Cause
+
+The affected employee was converted from a 1099 employee to a W-2 employee in Rippling. Due to the current application integration of Rippling into
```
Comment (cmmarslender) [inline 2022-01/2022-01-20-EmployeeCompromise.md:46]: incomplete?

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: 2022-01/2022-01-20-EmployeeCompromise.md (hunk 1 above)
Comment (pmaslana) [reply 2022-01/2022-01-20-EmployeeCompromise.md]: I intended that for admin employee, but substituted user. Its fixed now.

**Pull request closed without merging**